dev/com.ibm.ws.microprofile.opentracing.jaeger_fat/publish/servers/jaegerServerInventory/bootstrap.properties

```diff
@@ -0,0 +1,4 @@
+bootstrap.include=../testports.properties
+
+default.http.port=${bvt.prop.HTTP_secondary}
```

Comment:
I don't think you need to define them here.  They will be included in by the FAT infrastructure.

Reply:
It looks like these need to be defined for the Inventory app to work - otherwise, the error `CWWKG0075E: The value ${bvt.prop.HTTP_secondary} is not valid for attribute httpPort of configuration element httpEndpoint. The validation message was: Value "${bvt.prop.HTTP_secondary}" is not a number..` shows up. Removing `default.http.port` results in `Server returned HTTP response code: 500 for URL: http://localhost:8030/inventory/systems/localhost`

Reply:
You can simply use 

`private final int DEFAULT_PORT = Integer.valueOf(System.getProperty("bvt.prop.HTTP_secondary"));`

Reply:
Fixed.